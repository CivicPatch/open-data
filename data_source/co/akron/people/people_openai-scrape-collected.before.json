[
  {
    "name": "Brandon Hill",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "images": [],
    "roles": [
      {
        "data": "Mayor",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Mayor in the current governing members list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [],
    "websites": [],
    "start_dates": [],
    "end_dates": [
      {
        "data": "2026",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term end year explicitly stated as 2026.",
        "source": "https://townofakron.colorado.gov/government/board-of-trustees"
      }
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Jared Jefferson",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "images": [],
    "roles": [
      {
        "data": "Trustee",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Trustee in the current governing members list."
      },
      {
        "data": "Mayor Pro Tem",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Mayor Pro Tem in the current governing members list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [],
    "websites": [],
    "start_dates": [],
    "end_dates": [
      {
        "data": "2028",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term end year explicitly stated as 2028.",
        "source": "https://townofakron.colorado.gov/government/board-of-trustees"
      }
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Crystann Benson",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "images": [],
    "roles": [
      {
        "data": "Trustee",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Trustee in the current governing members list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [],
    "websites": [],
    "start_dates": [],
    "end_dates": [
      {
        "data": "2028",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term end year explicitly stated as 2028.",
        "source": "https://townofakron.colorado.gov/government/board-of-trustees"
      }
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Braden Brent",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "images": [],
    "roles": [
      {
        "data": "Trustee",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Trustee in the current governing members list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [],
    "websites": [],
    "start_dates": [],
    "end_dates": [
      {
        "data": "2028",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term end year explicitly stated as 2028.",
        "source": "https://townofakron.colorado.gov/government/board-of-trustees"
      }
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Ariella Gonzales-Vondy",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "images": [],
    "roles": [
      {
        "data": "Trustee",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Trustee in the current governing members list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [],
    "websites": [],
    "start_dates": [],
    "end_dates": [
      {
        "data": "2026",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term end year explicitly stated as 2026.",
        "source": "https://townofakron.colorado.gov/government/board-of-trustees"
      }
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Jennifer Hansen",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "images": [],
    "roles": [
      {
        "data": "Trustee",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Trustee in the current governing members list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [],
    "websites": [],
    "start_dates": [],
    "end_dates": [
      {
        "data": "2026",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term end year explicitly stated as 2026.",
        "source": "https://townofakron.colorado.gov/government/board-of-trustees"
      }
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "David Kembel",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "images": [],
    "roles": [
      {
        "data": "Trustee",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Trustee in the current governing members list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [],
    "websites": [],
    "start_dates": [],
    "end_dates": [
      {
        "data": "2026",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term end year explicitly stated as 2026.",
        "source": "https://townofakron.colorado.gov/government/board-of-trustees"
      }
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  }
]