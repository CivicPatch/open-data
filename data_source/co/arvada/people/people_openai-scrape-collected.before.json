[
  {
    "name": "Brad Rupert",
    "sources": [
      "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-",
      "https://www.arvada.org/158/About-City-Council"
    ],
    "images": [
      {
        "data": "https://www.arvada.org/ImageRepository/Document?documentId=4946",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Included in the group council photo.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "roles": [
      {
        "data": "Council Member At-Large",
        "llm_confidence": 0.95,
        "llm_confidence_reason": "Text states he was appointed to the at-large City Council seat, indicating role as Council Member At-Large."
      },
      {
        "data": "Council Member",
        "llm_confidence": 1,
        "llm_confidence_reason": "Listed as a councilmember in the group photo caption."
      }
    ],
    "divisions": [
      {
        "data": "At-Large",
        "llm_confidence": 0.95,
        "llm_confidence_reason": "The seat is described as an at-large City Council seat."
      },
      {
        "data": "At-Large",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated in the councilmembers list."
      }
    ],
    "phone_numbers": [],
    "emails": [
      {
        "data": "Council@arvada.org",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General City Council email provided for contact.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "websites": [
      {
        "data": "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "The URL provided is a profile/news page about Brad Rupert's appointment.",
        "source": "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-"
      },
      {
        "data": "https://www.arvada.org/158/About-City-Council",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "Page URL is the official City Council page.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "start_dates": [
      {
        "data": "2024-12-16",
        "llm_confidence": 0.95,
        "llm_confidence_reason": "Text states he took the Oath of Office on Dec. 16, 2024.",
        "source": "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-"
      }
    ],
    "end_dates": [
      {
        "data": "2025-11",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Term continues until the next regular election in November 2025.",
        "source": "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-"
      }
    ],
    "updated_at": "2025-06-27T20:12:55Z"
  },
  {
    "name": "Lauren Simpson",
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "images": [
      {
        "data": "https://www.arvada.org/ImageRepository/Document?documentId=4946",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Image titled 'Council Group Pic 2024' near the list of councilmembers including Lauren Simpson.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "roles": [
      {
        "data": "Mayor",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated as 'Lauren Simpson, Mayor' in the councilmembers list."
      }
    ],
    "divisions": [],
    "phone_numbers": [],
    "emails": [
      {
        "data": "Council@arvada.org",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General City Council email provided for contact.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "websites": [
      {
        "data": "https://www.arvada.org/158/About-City-Council",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "Page URL is the official City Council page.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-27T20:12:55Z"
  },
  {
    "name": "Randy Moorman",
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "images": [
      {
        "data": "https://www.arvada.org/ImageRepository/Document?documentId=4946",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Included in the group council photo.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "roles": [
      {
        "data": "Mayor Pro Tem",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated as 'Randy Moorman, Mayor Pro Tem, District 1' in the councilmembers list."
      },
      {
        "data": "Council Member",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "As Mayor Pro Tem is a council role, he is also a council member representing District 1."
      }
    ],
    "divisions": [
      {
        "data": "District 1",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated in the councilmembers list."
      }
    ],
    "phone_numbers": [],
    "emails": [
      {
        "data": "Council@arvada.org",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General City Council email provided for contact.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "websites": [
      {
        "data": "https://www.arvada.org/158/About-City-Council",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "Page URL is the official City Council page.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-27T20:12:55Z"
  },
  {
    "name": "John Marriott",
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "images": [
      {
        "data": "https://www.arvada.org/ImageRepository/Document?documentId=4946",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Included in the group council photo.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 1,
        "llm_confidence_reason": "Listed as a councilmember in the group photo caption."
      }
    ],
    "divisions": [
      {
        "data": "District 3",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated in the councilmembers list."
      }
    ],
    "phone_numbers": [],
    "emails": [
      {
        "data": "Council@arvada.org",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General City Council email provided for contact.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "websites": [
      {
        "data": "https://www.arvada.org/158/About-City-Council",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "Page URL is the official City Council page.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-27T20:12:55Z"
  },
  {
    "name": "Bob Fifer",
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "images": [
      {
        "data": "https://www.arvada.org/ImageRepository/Document?documentId=4946",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Included in the group council photo.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 1,
        "llm_confidence_reason": "Listed as a councilmember in the group photo caption."
      }
    ],
    "divisions": [
      {
        "data": "District 4",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated in the councilmembers list."
      }
    ],
    "phone_numbers": [],
    "emails": [
      {
        "data": "Council@arvada.org",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General City Council email provided for contact.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "websites": [
      {
        "data": "https://www.arvada.org/158/About-City-Council",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "Page URL is the official City Council page.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-27T20:12:55Z"
  },
  {
    "name": "Sharon Davis",
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "images": [
      {
        "data": "https://www.arvada.org/ImageRepository/Document?documentId=4946",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Included in the group council photo.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 1,
        "llm_confidence_reason": "Listed as a councilmember in the group photo caption."
      }
    ],
    "divisions": [
      {
        "data": "At-Large",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated in the councilmembers list."
      }
    ],
    "phone_numbers": [],
    "emails": [
      {
        "data": "Council@arvada.org",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General City Council email provided for contact.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "websites": [
      {
        "data": "https://www.arvada.org/158/About-City-Council",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "Page URL is the official City Council page.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-27T20:12:55Z"
  },
  {
    "name": "Shawna Ambrose",
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "images": [
      {
        "data": "https://www.arvada.org/ImageRepository/Document?documentId=4946",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Included in the group council photo.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 1,
        "llm_confidence_reason": "Listed as a councilmember in the group photo caption."
      }
    ],
    "divisions": [
      {
        "data": "District 2",
        "llm_confidence": 1,
        "llm_confidence_reason": "Explicitly stated in the councilmembers list."
      }
    ],
    "phone_numbers": [],
    "emails": [
      {
        "data": "Council@arvada.org",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General City Council email provided for contact.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "websites": [
      {
        "data": "https://www.arvada.org/158/About-City-Council",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "Page URL is the official City Council page.",
        "source": "https://www.arvada.org/158/About-City-Council"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-27T20:12:55Z"
  }
]