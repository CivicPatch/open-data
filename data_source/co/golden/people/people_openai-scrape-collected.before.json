[
  {
    "name": "Laura Weinberg",
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "images": [
      {
        "data": "https://www.cityofgolden.gov/set_site_up/government/2024_COG_LauraWeinberg_jmp_001_sm%20-%20Copy.jpg",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Image URL is a headshot near the name Laura Weinberg.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "roles": [
      {
        "data": "Mayor",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Mayor in the content."
      }
    ],
    "divisions": [],
    "phone_numbers": [
      {
        "data": "(303) 384-8000",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General city contact phone number provided; no direct personal phone found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "emails": [],
    "websites": [
      {
        "data": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "General City Council page linked; no personal website found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Lisa Vitry",
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "images": [
      {
        "data": "https://www.cityofgolden.gov/Images/Government/Document%20Center/2024_COG_LisaVitry_jmp_002_sm%20-%20web%20-%20Copy.jpg",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Image URL is a headshot near the name Lisa Vitry.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Listed as District One Councilor, which is a council member role."
      }
    ],
    "divisions": [
      {
        "data": "District One",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as District One Councilor."
      }
    ],
    "phone_numbers": [
      {
        "data": "(303) 384-8000",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General city contact phone number provided; no direct personal phone found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "emails": [],
    "websites": [
      {
        "data": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "General City Council page linked; no personal website found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Patty Evans",
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "images": [
      {
        "data": "https://www.cityofgolden.gov/Images/Government/Document%20Center/2024_COG_PattyEvans_jmp_002_sm_websquare.jpg",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Image URL is a headshot near the name Patty Evans.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Listed as District Two Councilor, which is a council member role."
      }
    ],
    "divisions": [
      {
        "data": "District Two",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as District Two Councilor."
      }
    ],
    "phone_numbers": [
      {
        "data": "(303) 384-8000",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General city contact phone number provided; no direct personal phone found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "emails": [],
    "websites": [
      {
        "data": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "General City Council page linked; no personal website found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Robert Reed",
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "images": [
      {
        "data": "https://www.cityofgolden.gov/Images/Government/City%20leadership/Reed-2022-400x533%20-%20Copy.jpg",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Image URL is a headshot near the name Robert Reed.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 0.95,
        "llm_confidence_reason": "Listed as Ward One Councilor and Mayor Pro-Tem, both primary governing roles."
      },
      {
        "data": "Mayor Pro Tempore",
        "llm_confidence": 0.95,
        "llm_confidence_reason": "Explicitly stated as Mayor Pro-Tem."
      }
    ],
    "divisions": [
      {
        "data": "Ward One",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Ward One Councilor."
      }
    ],
    "phone_numbers": [
      {
        "data": "(303) 384-8000",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General city contact phone number provided; no direct personal phone found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "emails": [],
    "websites": [
      {
        "data": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "General City Council page linked; no personal website found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Paul Haseman",
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "images": [
      {
        "data": "https://www.cityofgolden.gov/Images/Government/City%20leadership/Haseman-2022-400x533%20-%20Copy.jpg",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Image URL is a headshot near the name Paul Haseman.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Listed as Ward Two Councilor, a primary governing role."
      }
    ],
    "divisions": [
      {
        "data": "Ward Two",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Ward Two Councilor."
      }
    ],
    "phone_numbers": [
      {
        "data": "(303) 384-8000",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General city contact phone number provided; no direct personal phone found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "emails": [],
    "websites": [
      {
        "data": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "General City Council page linked; no personal website found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Don Cameron",
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "images": [
      {
        "data": "https://www.cityofgolden.gov/Images/Government/City%20leadership/2024_COG_DonCameron_jmp_002_sm-websquare.jpg",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Image URL is a headshot near the name Don Cameron.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Listed as Ward Three Councilor, a primary governing role."
      }
    ],
    "divisions": [
      {
        "data": "Ward Three",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Ward Three Councilor."
      }
    ],
    "phone_numbers": [
      {
        "data": "(303) 384-8000",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General city contact phone number provided; no direct personal phone found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "emails": [],
    "websites": [
      {
        "data": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "General City Council page linked; no personal website found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Bill Fisher",
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "images": [
      {
        "data": "https://www.cityofgolden.gov/Images/Government/City%20leadership/2024_COG_BillFisher_jmp_002_sm.jpg",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Image URL is a headshot near the name Bill Fisher.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "roles": [
      {
        "data": "Council Member",
        "llm_confidence": 0.9,
        "llm_confidence_reason": "Listed as Ward Four Councilor, a primary governing role."
      }
    ],
    "divisions": [
      {
        "data": "Ward Four",
        "llm_confidence": 1.0,
        "llm_confidence_reason": "Explicitly stated as Ward Four Councilor."
      }
    ],
    "phone_numbers": [
      {
        "data": "(303) 384-8000",
        "llm_confidence": 0.8,
        "llm_confidence_reason": "General city contact phone number provided; no direct personal phone found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "emails": [],
    "websites": [
      {
        "data": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
        "llm_confidence": 0.7,
        "llm_confidence_reason": "General City Council page linked; no personal website found.",
        "source": "https://www.cityofgolden.net"
      }
    ],
    "start_dates": [],
    "end_dates": [],
    "updated_at": "2025-06-28T18:43:47Z"
  }
]